fcoemon.c

```diff
@@ -3796,8 +3796,10 @@ int main(int argc, char **argv)
 		fcm_dcbd_init();
 
 	rc = fcm_srv_create(&srv_info);
-	if (rc != 0)
-		goto err_cleanup;
+	if (rc != 0) {
```

Comment:
How about "if (rc == EADDRINUSE)", since that's the case you are addressing, and the server setup can fail for other reasons.